kratos/tests/test_kratos_parameters.py

```diff
@@ -497,6 +497,17 @@ def test_remove_value(self):
         self.assertFalse(kp.Has("int_value"))
         self.assertFalse(kp.Has("level1"))
 
+    def test_remove_values(self):
+        kp = Parameters(json_string)
+        self.assertTrue(kp.Has("int_value"))
+        self.assertTrue(kp.Has("level1"))
+
+        list_remove = ["int_value", "level1"]
+        kp.RemoveValues(list_remove)

```

Comment:
Can you plz add a quick test for a failure too?

Reply:
I can...

Reply:
Done